Sources/Message+CloudKit.swift

```diff
@@ -16,17 +16,17 @@ extension Message {
 	static var dateKey: String { return "Date" }
 	
 	init?(cloudKitRecord: CKRecord) {
-		guard let messageText = cloudKitRecord["MessageText"] as? String,
-			date = cloudKitRecord.creationDate ?? (cloudKitRecord["Date"] as? NSDate)
+		guard let messageText = cloudKitRecord[Message.messageTextKey] as? String,
```

Comment:
Thanks for fixing this. I've run into this everytime I've done Bulletin Board as a guided project, but keep forgetting to actually fix it in the solution.